backend/regalloc/regalloc_gi_utils.ml

```diff
@@ -75,13 +84,24 @@ module Selection_heuristics = struct
     | First_available
     | Best_fit
     | Worst_fit
+    | Random_for_testing
 
-  let all = [First_available; Best_fit; Worst_fit]
+  let all = [First_available; Best_fit; Worst_fit; Random_for_testing]
 
   let to_string = function
     | First_available -> "first_available"
     | Best_fit -> "best_fit"
     | Worst_fit -> "worst_fit"
+    | Random_for_testing -> "random"
+
+  let include_in_random = function
+    | Random_for_testing | Worst_fit -> false
+    | _ -> true
```

Comment:
Please avoid the catch-all, there
is a good chance we will add
more constuctors.

Reply:
:+1: